addon/components/lazy-image.js

```diff
@@ -19,6 +19,15 @@ export default Component.extend(InViewportMixin, ImageLoadMixin, {
 
   classNames: ['lazy-image-container'],
 
+  concatenatedProperties: ['imgClassNames'],
+
+  imgClassNames: ['lazy-image'],
```

Comment:
`class` is reserved keyword and using it results in `SyntaxError: Unexpected token`